release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1849,17 +1836,19 @@ With this release, availability sets are no longer modified after creation, allo
 
 * Previously, when installing a cluster on {vmw-full}, installations could fail if the API and Ingress server virtual IPs were outside of the machine network. With this update, the installation program includes the API and Ingress server virtual IPs in the machine network by default. If you specify the API and Ingress server virtual IPs, ensure that they are in the machine network. (link:https://issues.redhat.com/browse/OCPBUGS-36553[OCPBUGS-36553])
 
-[discrete]
-[id="ocp-release-note-insights-operator-bug-fixes_{context}"]
-==== Insights Operator
+* Previously, when installing a cluster on {ibm-power-server-title}, the installation failed if you selected the Madrid zone due to an image import error. With this update, the installation program has been modified to use the correct storage bucket name and continue the installation successfully. (link:https://issues.redhat.com/browse/OCPBUGS-50899[OCPBUGS-50899])
 
-[discrete]
-[id="ocp-release-note-kube-controller-bug-fixes_{context}"]
-==== Kubernetes Controller Manager
+* Previously, when destroying a cluster installed on {ibm-power-server-title}, some resources including network subnets were not deleted. With this update, all network resources are deleted when the cluster is destroyed. (link:https://issues.redhat.com/browse/OCPBUGS-50657[OCPBUGS-50657])
 
-[discrete]
-[id="ocp-release-note-kube-scheduler-bug-fixes_{context}"]
-==== Kubernetes Scheduler
+* Previously, when installing a cluster using the Assisted Installer, the installation could fail due to a timeout when pulling images. With this update, the timeout has been increased so that the installation program completes pulling images. link:https://issues.redhat.com/browse/OCPBUGS-50655[OCPBUGS-50655]
+
+* Previously, in some slower PrismCentral environments, the installation program would fail with a timeout when make the prism-api call to load the {op-system} image. The timeout value was previously 5 minutes. With this release, the prism-api call timeout value is a configurable parameter in the `install-config.yaml` file as `platform.nutanix.prismAPICallTimeout`, with a default value of 10 minutes. (link:https://issues.redhat.com/browse/OCPBUGS-48570[OCPBUGS-48570])
```

Comment:
```suggestion
* Previously, in some slower Prism Central environments, the installation program would fail with a timeout when make the `prism-api` call to load the {op-system} image. The timeout value was previously 5 minutes. With this release, the `prism-api` call timeout value is a configurable parameter in the `install-config.yaml` file as `platform.nutanix.prismAPICallTimeout`, with a default value of 10 minutes. (link:https://issues.redhat.com/browse/OCPBUGS-48570[OCPBUGS-48570])
```